src/style.css

```diff
@@ -0,0 +1,80 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir-Heavy";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 800;
+  font-style: normal;
+}
+
+* {
+  font-family: Avenir, "Avenir-Heavy", sans-serif;
+  box-sizing: border-box;
+  font-weight: 300;
+}
```

Comment:
always set font styles to the `html` tag. In `*` can be used only `box-sizing` anything else is forbidden